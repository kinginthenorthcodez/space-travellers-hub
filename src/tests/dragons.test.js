import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Dragons from '../components/dragons';

// Dragons component tests
describe('Dragons snapshot', () => {
  afterEach(cleanup);

  it('should render dragons correctly', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <Dragons />
      </Provider>,
    );
    expect(asFragment(<Dragons />)).toMatchSnapshot();
  });
});
