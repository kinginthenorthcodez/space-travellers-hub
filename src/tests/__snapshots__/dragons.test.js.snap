// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dragons snapshot should render dragons correctly 1`] = `
<DocumentFragment>
  <div
    class="dragon-section"
  >
    <hr
      class="line"
    />
  </div>
</DocumentFragment>
`;
